tests/check_e2e.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+if [ "x$TRAVIS_BUILD_DIR" == "x" ]; then
+  export TRAVIS_BUILD_DIR=$(cd $(dirname "$0")/../ && pwd);
```

Comment:
Why? Changing Travis environment variables isn't a good idea 🤔

Reply:
Yep, I reverted this change.